mapflow/functional/view/data_catalog_view.py

```diff
@@ -335,15 +345,20 @@ def clear_mosaic_info(self):
     def show_image_info(self, image: ImageReturnSchema):
         if not image:
             return
+        pixel_size = sum(list(image.meta_data.values())[6])/len(list(image.meta_data.values())[6])
```

Comment:
Won't it be better to make metadata a dataclass / schema as well, and not just dict? Because this addressing by index looks not good